Improve test coverage of sink values - part2.

ziransun opened the pull request:
#### bcace83dfe07692e49565696b881a1cd682a8776
<pre>
Improve test coverage of sink values - part2.
<a href="https://bugs.webkit.org/show_bug.cgi?id=273763">https://bugs.webkit.org/show_bug.cgi?id=273763</a>

Reviewed by Tim Nguyen.

Add checks on sink values.

* LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy-callback-arguments.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy-report-only.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy.html:

Specifically for default policy test files.

Canonical link: <a href="https://commits.webkit.org/278804@main">https://commits.webkit.org/278804@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/846ee2e225d07f587e7fb5d6121959f24f49b816

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/51286 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/30592 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/3628 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/54543 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/1976 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/36903 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/1655 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/41767 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/53385 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/28236 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/44217 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/22885 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/25561 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/1475 "Passed tests") | [⏳ 🛠 wpe-skia ](https://ews-build.webkit.org/#/builders/WPE-Skia-Build-EWS "Waiting in queue, processing has not started yet") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/47530 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/1548 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/56139 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/26401 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/1443 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/49166 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/27644 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/44280 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/48326 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/11296 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/28534 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/27379 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy-callback-arguments.html:
```diff
@@ -17,6 +17,17 @@
+        case TrustedScript:
+        assert_equals(args[2], 'HTMLScriptElement text');
+        break;
+      }
```
Can you consistently indent TrustedScript?

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy-report-only.html:
```diff
@@ -65,8 +65,20 @@
+      case TrustedScript:
+       assert_equals(sink, 'HTMLScriptElement text');
+       break;
+    }
```
Ditto here

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy.html:
```diff
@@ -65,8 +65,20 @@
+      case TrustedScript:
+       assert_equals(sink, 'HTMLScriptElement text');
+       break;
+    }
```
Ditto here

ziransun committed:
Improve test coverage of sink values - part2.
https://bugs.webkit.org/show_bug.cgi?id=273763

Reviewed by Tim Nguyen.

Add checks on sink values.

* LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy-callback-arguments.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy-report-only.html:
* LayoutTests/imported/w3c/web-platform-tests/trusted-types/default-policy.html:

Specifically for default policy test files.

Canonical link: https://commits.webkit.org/278804@main
